lib/osnetwork.rb

```diff
@@ -0,0 +1,58 @@
+require 'clamp'
+require_relative 'osproject'
+require_relative 'osproject_ios'
+require_relative 'osproject_android'
+
+class NetworkHandler
+    @instance = new
+
+    private_class_method :new
+
+    def self.instance
+      @instance
+    end
+
+    def track_uri
+      URI.parse('https://api.onesignal.com/api/v1/track')
```

Comment:
Maybe the url can be defined as a const as well.

Reply:
Also, maybe you can move this as part of the `getHttp_net` method, and receive the `url` as param instead of the `uri`. What do you think?